src/proteus/dswx_hls.py

```diff
@@ -508,31 +665,6 @@ def get_dswx_hls_cli_parser():
                         help='Product ID that will be saved in the output'
                         "product's metadata")
 
-    parser.add_argument('--use-otsu-terrain-masking',
-                        dest='flag_use_otsu_terrain_masking',
-                        action='store_true',
-                        default=None,
-                        help=('Compute and apply terrain masking using Otsu'
-                              ' thresholding'))
-
-    parser.add_argument('--min-slope-angle',
-                        dest='min_slope_angle',
-                        type=float,
-                        help='')
-
-    parser.add_argument('--max-sun-local-inc-angle',
-                        dest='max_sun_local_inc_angle',
-                        type=float,
-                        help='Maximum local-incidence angle')
-
-    parser.add_argument('--mask-adjacent-to-cloud-mode',
-                        dest='mask_adjacent_to_cloud_mode',
-                        type=str,
-                        choices=['mask', 'ignore', 'cover'],
-                        help='Define how areas adjacent to cloud/cloud-shadow'
-                        ' should be handled. Options: "mask", "ignore", and'
-                        ' "cover"')
-
```

Comment:
Why are we removing all these parameters? As we discussed offline, we should be able to update the code to use them properly.